telephoneDirectory/telephoneDirectory/directory.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+
+#define MAX_LEN_NAME 100
+#define PHONE_LENGHT 18
+#define YOUR_FILE "..\\directory.txt"
+
+typedef struct {
+	char name[MAX_LEN_NAME];
+	char phone[PHONE_LENGHT];
+} PhoneBookEntry;
+
+int newEntry(PhoneBookEntry* book, int numberOfNewEntries, int numberOfFilesEntries) {
+	int index = numberOfNewEntries + numberOfFilesEntries;
+	char surname[MAX_LEN_NAME];
```

Comment:
```suggestion
	char surname[MAX_LEN_NAME] = {'\0'};
```
Потому что неинициализированные переменные запрещены стайлгайдом